llvm/include/llvm/IR/IntrinsicsRISCV.td

```diff
@@ -1736,6 +1736,17 @@ let TargetPrefix = "riscv" in {
                                               [llvm_anyptr_ty, LLVMScalarOrSameVectorWidth<0, llvm_i1_ty>,
                                                llvm_anyint_ty]),
                                   [NoCapture<ArgIndex<nf>>, IntrWriteMem]>;
+
+    // Input: (<stored values>..., pointer, offset, mask, vl)
```

Comment:
I didn't catch this in the previous review, but "offset" should probably be "stride" for clarity.